### ISSUE TITLE
dark mode 

### ISSUE THREAD

**Issue by lazyb5** [Jerrkawz/google-voice-desktop-app#2]
I would like to a dark mode user style added. thanks

**Comment by Jerrkawz** [comment]
Yeah i like that idea. Ill probably take it a step further and just make it themeable by any one and include one dark theme by default.

**Comment by brmonaghan** [comment]
Whats the status on this @Jerrkawz ? I can develop the dark theme and do another PR for you!

**Comment by brmonaghan** [comment]
Unfortunately GV is only in light mode, so this idea is virtually impossible. @Jerrkawz since this cannot be done, this can be closed.

**Comment by lazyb5** [comment]
This is not impossible. I'm pretty sure you can elect css into an electron app easily. Even simpler way would be if your can toggle the force dark web content flag in chrome.

**Comment by brmonaghan** [comment]
I tried that and it's still light. See #15 for details.

**Comment by brmonaghan** [comment]
> This is not impossible. I'm pretty sure you can elect css into an electron app easily. Even simpler way would be if you can toggle the force dark web content flag in chrome.

Wait, CSS can be implemented into electron apps? If so, how do you do this?

**Comment by Jerrkawz** [comment]
Im currently working on the theming, its quite complex but when I'm finished it will be a system capable of theming the main components inside the voice app with different color palettes, planning on shipping with 4-5 different themes. Join discord for screenshots and progress
